supplementary_style_guide/glossary_terms_conventions/general_conventions/s.adoc

```diff
@@ -720,7 +720,7 @@ Therefore, if you need to use the indefinite article before "sosreport", use _an
 // OCP: Added "In Red Hat OpenShift,"
 [[spec]]
 ==== image:images/yes.png[yes] spec (noun)
-*Description*: In Red Hat OpenShift, in addition to "spec file", which is permitted when it relates to RPM spec files, you can also use "spec" for general usage when you describe Kubernetes or OpenShift Container Platform object specs, manifests, or definitions.
+*Description*: In Red Hat OpenShift, in addition to "spec file", which is permitted when it relates to an RPM spec file, you can also use "spec" for general usage when you describe Kubernetes or OpenShift Container Platform object specs, manifests, or definitions.
```

Comment:
Run-on. Break up into two sentences.

Reply:
Does the following revision work:

> In Red Hat OpenShift, "spec and "spec file" are permitted when these terms relate to an RPM spec file. You can also use "spec" for general usage when you describe Kubernetes or OpenShift Container Platform object specs, manifests, or definitions.

Reply:
"In Red Hat OpenShift", otherwise looks good!

Reply:
Thanks. Updated!